backend/src/exceptions/files-error/files-error.exception.ts

```diff
@@ -0,0 +1,24 @@
+import { HttpError } from 'guruhub-shared/exceptions/exceptions';
+
+import {
+  CustomExceptionName,
+  ExceptionMessage,
+  HttpCode,
+} from '~/common/enums/enums';
+
+type Constructor = {
+  message?: ExceptionMessage | string;
+  status?: HttpCode | number;
```

Comment:
```suggestion
  message?: ExceptionMessage;
  status?: HttpCode;
```
remove from all numbers. should be only `HttpCode` and `ExceptionMessage`